rescript/trim_alignment.py

```diff
@@ -190,11 +190,13 @@ def _trim_alignment(expand_alignment_action,
                     primer_fwd=None,
                     primer_rev=None,
                     position_start=None,
-                    position_end=None) -> AlignedDNAFASTAFormat:
+                    position_end=None,
+                    n_threads=1) -> AlignedDNAFASTAFormat:
     """
     Trim alignment based on primer alignment or explicitly specified
     positions. When at least one primer sequence is given, primer-based
-    trimming will be performed, otherwise position-based trimming is done.
+    trimming will be performed via the `addfragments` option of mafft,
+    otherwise position-based trimming is done.
```

Comment:
the `n_threads` param would not be effective as far as I can tell when position-based trimming is done, so this should be mentioned in the param description

Reply:
Oooh, good catch!